models/enhancement_additions/int_zendesk__comment_metrics.sql

```diff
@@ -0,0 +1,41 @@
+with ticket_comments as (
```

Comment:
I agree that these should be related and we can condense a number of the comment models. I have since used your suggestion and created a `int_zendesk_comments_enriched` model which the downstream models reference. Additionally, I was able to remove the one touch resolution model into the `int_zendesk__comment_metrics` model.

Let me know if you want to discuss any of these updates.